.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+name: lint
+description: "Check linting on pull requests"
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)